eng/Baseline.xml

```diff
@@ -10,7 +10,6 @@ build of ASP.NET Core 2.2.x. Update this list when preparing for a new patch.
   <Package Id="dotnet-user-secrets" Version="2.2.0" />
   <Package Id="dotnet-watch" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.Antiforgery" Version="2.2.0" />
-  <Package Id="Microsoft.AspNetCore.ApiAuthorization.IdentityServer" Version="2.2.0-preview-35687" />
```

Comment:
Revert this change

Reply:
Why do we keep this if we didn't ship this.